File_handling/index.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
```

Comment:
Delete the duplicate files. Probably you want to keep only the js files from root, right?